hudi-common/src/main/java/org/apache/hudi/common/table/timeline/CompletionTimeQueryView.java

```diff
@@ -16,27 +16,33 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.client.timeline;
+package org.apache.hudi.common.table.timeline;
 
 import org.apache.hudi.common.table.HoodieTableMetaClient;
-import org.apache.hudi.common.table.timeline.HoodieArchivedTimeline;
-import org.apache.hudi.common.table.timeline.HoodieInstant;
-import org.apache.hudi.common.table.timeline.HoodieTimeline;
 import org.apache.hudi.common.util.Option;
 
 import org.apache.avro.generic.GenericRecord;
 
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Date;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
 import static org.apache.hudi.common.table.timeline.HoodieArchivedTimeline.COMPLETION_TIME_ARCHIVED_META_FIELD;
 import static org.apache.hudi.common.table.timeline.HoodieTimeline.EQUALS;
 import static org.apache.hudi.common.table.timeline.HoodieTimeline.GREATER_THAN;
+import static org.apache.hudi.common.table.timeline.HoodieTimeline.GREATER_THAN_OR_EQUALS;
+import static org.apache.hudi.common.table.timeline.HoodieTimeline.LESSER_THAN;
 
 /**
  * Query view for instant completion time.
  */
-public class CompletionTimeQueryView implements AutoCloseable {
+public class CompletionTimeQueryView implements AutoCloseable, Serializable {
+  private static final long serialVersionUID = 1L;
+
+  private static final long MILLI_SECONDS_IN_THREE_DAYS = 3 * 24 * 3600 * 1000;
```

Comment:
Do we need some internal config to control this?

Reply:
Currently 3 days should work well, we can make it configurable if it is necessary.

Reply:
let's track this in JIRA?

Reply:
see https://issues.apache.org/jira/browse/HUDI-6929